arch/ua_network_tcp.c

```diff
@@ -608,7 +608,7 @@ UA_StatusCode UA_ClientConnectionTCP_poll(UA_Client *client, void *data) {
 
     if(clientsockfd == UA_INVALID_SOCKET) {
             UA_LOG_WARNING(UA_Log_Stdout, UA_LOGCATEGORY_NETWORK,
-                            "Could not create client socket: %s", strerror(UA_ERRNO));
+                            "Could not create client socket: %s", UA_ERRNO);
```

Comment:
This should still be strerror. If that's not supported in WEC, then redefine strerror in your architecture file